rollup.config.js

```diff
@@ -4,18 +4,19 @@ import sourceMaps from 'rollup-plugin-sourcemaps';
 import typescript from 'rollup-plugin-typescript2';
 import json from 'rollup-plugin-json';
 
-export default outputFile => ({
-  input: 'src/index.ts',
+export default (outputFile, inputFile = 'src/index.ts', external) => ({
```

Comment:
I wonder if there's a better way to extend/overwrite the default configuration. Perhaps the options should be an object? Or add a second `options` arg that is merged with the default config and can overwrite it?

Reply:
The latter seems like the cleaner approach to me.